tg/controllers.py

```diff
@@ -495,6 +499,31 @@ def edit_msg(self) -> None:
                     self.model.edit_message(text=text)
                     self.present_info("Message edited")
 
+    @bind(chat_handler, ["dd"])
+    def leave_chat(self) -> None:
+        """Leave group or cloase private, secret chat"""
+        # can_be_deleted_only_for_self
+        # can_be_deleted_for_all_users
+        chat = self.model.chats.chats[self.model.current_chat]
+        resp = self.view.status.get_input(
+            "Are you sure you want to delete the chat?[y/N]"
+        )
+        if resp is None or is_no(resp):
```

Comment:
` resp is None ` is redundant

Reply:
None returns when input was cancelled. If we are at the second question then we can’t decline chat deletion. Escape will reply to no instead of canceling deletion

Reply:
ok, but ` is_no(resp)` would be enough, because `is_no` is able to work with `None`, isn't it?